### DISCUSSION TITLE
Ola, como puedo contribuir a esto?

### DISCUSSION THREAD

**Discussion by mariofalco** [BreakBuddy/BreakBuddy#13] [Q&A] [answered]
Ola, como puedo contribuir a esto? Soy un desarrollador de España. Tengo 5 años de experiencia en nodejs. Puedo ayudar con cualquier cosa si creas un problema abierto para mí, gracias.

**Comment by hursittarcan** [comment, marked as answer]
Hey @mariofalco, 
Your help would be much appreciated 🙌 
We have a few open issues atm, most of them focus on adding extra entries to our 'database'. If you are intrested in doing one/or more you are free to do so. 

Don't hesitate to contact me if you have any more questions! 
Thanks again ❤️

**Comment by zizooom** [comment]
Thanx mmmu
